blockchain/blockchain.go

```diff
@@ -630,11 +631,16 @@ func (bc *blockchain) MintNewBlock(
 	if newblockHeight == 1 {
 		prevBlkHash = bc.config.Genesis.Hash()
 	}
+	var bloom block.BloomFilter
+	if newblockHeight >= genesis.PacificHeight() {
```

Comment:
add in bloom filter for height >= pacificHeight

Reply:
get the height from config instead of reading the default value.

Let's define a new height as 0.6.x is released. For example, "CaliforniaHeight"?